src/components/devices/sigenergy/device.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+	<div class="device-sigenergy">
+		<openwb-base-heading>
+			Einstellungen für Sigenergy
+			<span class="small">(Modul: {{ $options.name }})</span>
+		</openwb-base-heading>
```

Comment:
Hier könnte noch ein Hinweis hin, dass für das Lastmanagement keine Ströme auslesbar sind.

Reply:
Das stimmt meiner Meinung nach nicht. Der Zähler ermittelt Leistungen je Phase. in `component_state.py` werden daraus (unter Annahme von 230V auf den Phasen) Ströme berechnet. Damit sollte das Lastmanagement (zumindest näherungsweise) arbeiten können.